geomstats/information_geometry/normal.py

```diff
@@ -247,3 +247,55 @@ def metric_matrix(base_point=None):
         if metric_mat.ndim == 3 and metric_mat.shape[0] == 1:
             return metric_mat[0]
         return metric_mat
+
+    def sectional_curvature(self, tangent_vec_a, tangent_vec_b, base_point=None):
+        r"""Compute the sectional curvature.
+
+        In the literature sectional curvature is noted K.
+        For two orthonormal tangent vectors :math:`x,y` at a base point,
+        the sectional curvature is defined by :math:`K(x,y) = <R(x, y)x, y>`.
+        For non-orthonormal vectors, it is
+        :math:`K(x,y) = <R(x, y)x, y> / \\|x \wedge y\\|^2`.
```

Comment:
The convention taken by do Carmo for the curvature is not the one chosen in the rest of geomstats. So the formula should be (rewriting also the denominator might be more clear)
`K(x,y) = <R(x, y)y, x> / (<x, x><y, y> - <x, y>^2)`

Reply:
Thanks, I've adapted the docstring